assembl/graphql/view.py

```diff
@@ -13,18 +15,40 @@
 class LoggingMiddleware(object):
     def resolve(self, next, source, gargs, context, info, *args, **kwargs):
         if source is None:
-            variables = {}
+            modified_variables = {}
             for key, value in info.variable_values.items():
```

Comment:
what about just copying the dict and deleting password?

Reply:
Keeping the key tells me that the user tried to change her password. We lose this info if you delete the key.